src/main/java/mate/academy/springboot/datajpa/service/h2/CategoryServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.springboot.datajpa.service.h2;
+
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CategoryServiceImpl implements CategoryService {
+    private final CategoryRepository categoryRepository;
+
+    @Autowired
```

Comment:
it's not consistent, use constructor or @RequiredArgsConstructor, but not both of them